### DISCUSSION TITLE
react build - t is not defined

### DISCUSSION THREAD

**Discussion by jmseo111** [GrapesJS/grapesjs#5591] [Q&A]
plzzzzzzz HELP ME !!!!!!!!!!!!

currentlt, i'm using REACT.
it works well at LOCAL environment. has NO ERROR.
but when i build, REFERENCE ERROR : t is not defined.

how can i solve it...?
i can't figure out what's the reason is.....

![image](https://github.com/GrapesJS/grapesjs/assets/76142698/2e66ec00-db72-4d77-8a41-c0ea4d8c4796)

**Comment by artf** [comment]
Looks like an error from a custom grapesjs Component with JS, can you show the source of that component?
